### ISSUE TITLE
[7.2.0] Conflicting outputs bug with `injectable_generator `

### ISSUE THREAD

**Issue by mariopepe** [Milad-Akarie/auto_route_library#1558]
I have updated to `auto_route: 7.1.0` and `auto_route_generator: 7.1.0` and now I am getting this bug:

```console
[SEVERE] Conflicting outputs
Both injectable_generator:injectable_config_builder and
auto_route_generator:auto_router_module_generator may output lib/bootstrap.module.dart.
Potential outputs must be unique across all builders.
See https://github.com/dart-lang/build/blob/master/docs/faq.md#why-do-builders-need-unique-outputs
```

---

This is my full `pubspec.yaml`:

```yaml
environment:
  sdk: '>=3.0.0 <4.0.0'

dependencies:
  adaptive_dialog: ^1.8.3+1
  add_to_cart_animation: ^2.0.3
  appinio_social_share:
    path: local_packages/appinio_social_share
  auto_route: 7.1.0
  auto_size_text: ^3.0.0
  bloc: ^8.1.2
  carousel_slider: ^4.2.1
  cloud_firestore: ^4.7.1
  cloud_functions: ^4.3.1
  collection: ^1.17.2
  country_code_picker:
    path: local_packages/country_code_picker
  cupertino_icons: 1.0.5
  dartz: ^0.10.1
  dropdown_search: 5.0.5
  encrypt: ^5.0.1
  firebase_analytics: ^10.4.1
  firebase_auth: ^4.6.1
  firebase_core: ^2.13.0
  firebase_crashlytics: ^3.3.1
  firebase_dynamic_links: ^5.3.1
  firebase_in_app_messaging: ^0.7.3+1
  firebase_messaging: ^14.6.1
  firebase_remote_config: ^4.2.1
  firebase_storage: ^11.2.1
  flutter:
    sdk: flutter
  flutter_bloc: ^8.1.3
  flutter_contacts: ^1.1.6
  flutter_email_sender: ^5.2.0
  flutter_local_notifications: ^14.1.0
  flutter_native_splash: ^2.3.0
  flutter_sms: ^2.3.3
  flutter_spinkit: ^5.2.0
  flutter_svg: ^2.0.6
  freezed_annotation: 2.2.0
  geocoding: ^2.1.0
  geolocator: ^9.0.2
  get_it: ^7.6.0
  google_mobile_ads: ^3.0.0
  http: ^0.13.6
  image: ^4.0.17
  image_cropper: ^4.0.1
  image_gallery_saver: 1.7.1
  image_picker: ^0.8.7+5
  in_app_review: ^2.0.6
  injectable: ^2.1.1
  intl: ^0.18.1
  introduction_screen: ^3.1.8
  isar: ^3.1.0+1
  isar_flutter_libs: ^3.1.0+1
  jiffy: ^6.1.0
  json_annotation: ^4.8.1
  lottie: ^2.3.2
  modal_bottom_sheet: ^3.0.0-pre
  package_info_plus: ^4.0.1
  path: ^1.8.3
  path_provider: ^2.0.15
  permission_handler: 10.2.0
  pinput: ^2.2.31
  provider: ^6.0.5
  purchases_flutter: ^4.12.0
  quickalert: ^1.0.1
  rxdart: 0.27.7
  screenshot: ^2.1.0
  share_plus: ^7.0.1
  shared_preferences: ^2.1.1
  shimmer: ^3.0.0
  sized_context: ^1.0.0+4
  social_share: ^2.3.1
  timezone: ^0.9.2
  url_launcher: ^6.1.11
  url_strategy: ^0.2.0
  uuid: 3.0.7
  very_good_analysis: ^5.0.0+1

dev_dependencies:
  auto_route_generator: 7.0.0
  build_runner: ^2.4.4
  flutter_launcher_icons: ^0.13.1
  flutter_lints: 2.0.1
  freezed: ^2.3.4
  injectable_generator: ^2.1.5
  intl_translation: ^0.18.2
  isar_generator: ^3.1.0+1
  json_serializable: ^6.7.0

dependency_overrides:
  collection: ^1.17.2
  frontend_server_client: 3.2.0
  macos_ui:
    git:
      url: https://github.com/Mayb3Nots/macos_ui.git
      ref: dev
  visibility_detector: ^0.4.0+2
```

**Comment by Milad-Akarie** [comment]
@mariopepe fix will be pushed soon

**Comment by Milad-Akarie** [comment]
@mariopepe  fixed in auto_route_generator 7.1.1

**Comment by mariopepe** [comment]
The best!!!
